recipes/pp-sketchlib/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "pp-sketchlib" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/johnlees/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: cc78e11f7afef7128fed4b048ccdf44e968825fde257d5b25925f9fb68a7c32a
+  patches:
+    - 0001-Replace-cmake-with-make.patch  # [linux64 and cuda_compiler_version != "None"]
+    - 0002-Pass-libloc-to-make-command.patch  # [linux64 and cuda_compiler_version != "None"]
+    - 0003-Fix-target-dependencies-in-Makefile.patch  # [linux64 and cuda_compiler_version != "None"]
+
+build:
+  number: 0
+  skip: True  # [py2k or win]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+  missing_dso_whitelist:
+    - "*/libcuda.*"
+  entry_points:
+    - poppunk_sketch = pp_sketch.__main__:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux64]
+    - cmake
+  host:
+    - python
+    - pip
+    - pybind11
+    - highfive
+    - zlib
+    - eigen
+    - armadillo
+    - hdf5
+    - openblas
+  run:
+    - python
+    - numpy
+    - zlib
+    - hdf5
+    - h5py
+    - openblas
+    - libgfortran-ng
+    - libgfortran  # [osx]
```

Comment:
We need to check which of the host deps have run exports. Then we should remove any host deps that do since they insert the proper runtime requirement.

Reply:
Look through and seems we can remove `hdf5` and `openblas` which have run exports. I've also added `cudatoolkit` and `armadillo` to ignore their run exports, as the former is linked statically, and the latter just uses the includes when building and not the library when running

Reply:
You need to follow the docs on using blas and not list openblas directly.

Reply:
At the moment the linker line uses `-lopenblas` explicitly. Would it be better to make `openblas` only with:
```
requirements:
  host:
    - openblas
  run:
    - libblas * *openblas
    - openblas
```
or to change the linker line to `-lblas` and have:
```
requirements:
  host:
    - libblas
```

Reply:
(referring to https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=blas)

Reply:
What to support is up to you. Just make sure to follow the docs so that things are setup properly with the ecosystem.

Reply:
I had not realized we had the openblas-only option in the docs, so that is good by me.

Reply:
Great, I have gone for that